src/views/Dashboard.vue

```diff
@@ -186,6 +193,10 @@ export default {
     /** The Incorporation Application's Temporary Registration Number string. */
     tempRegNumber (): string {
       return sessionStorage.getItem('TEMP_REG_NUMBER')
+    },
+
+    visibleForStaff (): boolean {
+      return this.isRoleStaff
```

Comment:
Do you think we need a feature flag to limit how this feature is deployed? Or do you think this feature will be self-complete and can be deployed through Dev-Test-Prod normally (ie, in sync with API changes)?

Reply:
I've been thinking of it... maybe it could be a good because if it's added and the other ticket to display those filings is not complete, it may be a problem, right?

Reply:
OK, use this new FF. Default it to False.
![image](https://user-images.githubusercontent.com/10002889/119053905-97925e00-b97b-11eb-8e9a-637a23198961.png)

Reply:
Actually, you'll want to restore `visibleForStaff()` in order to check the FF as well.

Example: https://github.com/bcgov/namerequest/blob/9b3ff47e5f8b0a040b3764e6358603d06bed87fb/src/components/existing-request/existing-request-search.vue#L165

Reply:
ok. Added.